backend/backend/settings.py

```diff
@@ -138,6 +139,8 @@
 # https://docs.djangoproject.com/en/4.1/howto/static-files/
 
 STATIC_URL = 'static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+STATICFILES_DIRS = (os.path.join(BASE_DIR, 'static'),)
```

Comment:
When running any Docker command I get an error, that says: `(staticfiles.E002) The STATICFILES_DIRS setting should not contain the STATIC_ROOT setting.`.
This could be the solution: https://stackoverflow.com/a/27213753 (Just using an different directory for the `STATIC_ROOT`)

```suggestion
STATIC_ROOT = os.path.join(BASE_DIR, 'assets')
STATICFILES_DIRS = (os.path.join(BASE_DIR, 'static'),)
```